### ISSUE TITLE
Remove drupal as a depency

### ISSUE THREAD

**Issue by mikkeschiren** [webflo/packagist#13]
For years we have been working with profiles for developing drupal sites, which fit in perfectly with the workflow with aegir, which we have adapt to, and now we would like to step away from drush make and move to composer and the common php-way, which works quite well, except that when downloading modules to a profile using your packagist, drupal core as added as an dependency, which if strange if we have our composer file in drupal/profiles/myprofile - we need need to remove the drupal core folder to make it work.

So we would really like to remove drupal as an dependency from webflo's packagist, or add an option in the composer.json to not download drupal core, to make it more usable, and more, drupal-like.

**Comment by derhasi** [comment]
If you are using composer, you are supposed to use a [composer.json for the whole project](https://github.com/derhasi/drupal-composer-project/blob/master/composer.json). Nontheless, if you want to avoid certain packages to be downloaded due to your setup, you could simply add the following to your composer.json:

``` json
  "replace": {
    "drupal/drupal": "*"
  },
```

Btw: we headed over to https://github.com/drupal-composer/drupal-packagist and https://github.com/drupal-composer/drupal-parse-composer

**Comment by mikkeschiren** [comment]
Yes, I am using for the whole project, but we work with profiles as separate repos, so the replace solution works fine, thanks :-)
